server/configureChangeFeeds/userEmailUpdated.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-console, camelcase */
+import processChangeFeedWithAutoReconnect from 'rethinkdb-changefeed-reconnect'
+
+import {connect} from 'src/db'
+import handleConnectionError from './util/handleConnectionError'
+
+const r = connect()
+
+export default function userEmailUpdated(queueService) {
+  processChangeFeedWithAutoReconnect(
+    _getFeed,
+    _getFeedProcessor(queueService),
+    handleConnectionError,
+    {changefeedName: 'email updated'}
+  )
+}
+
+function _getFeed() {
+  return r.table('users')
+    .changes()
+    .filter(
+      r.row('old_val')('email')
```

Comment:
For new inserts, this will cause an error as the value of `old_val` will be null.

Recommend:
```js
return r.table('users')
    .changes()
    .filter(
      r.row('old_val').default({})('email')
        .ne(
          r.row('new_val')('email')
        )
    )
```